assets/extensions/login-template.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<!--
+
+This template can be modified and used to customize the login page. To replace
+the login page, set master configuration option oauthConfig.templates.login to
+the path of the template file. Do not change lines with curly braces {{ }} below.
```

Comment:
Ha, adding the empty curly braces in the commend causes template execution to fail...